src/Capgemini.PowerApps.PackageDeployerTemplate/Capgemini.PowerApps.PackageDeployerTemplate.csproj

```diff
@@ -20,7 +20,7 @@
   </PropertyGroup>
   <ItemGroup>
     <None Include="..\..\.editorconfig" Link=".editorconfig" />
-    <None Include="images\Capgemini.PowerApps.PackageDeployerTemplate.svg" Pack="true" PackagePath="" />
+    <None Include="images\Capgemini.PowerApps.PackageDeployerTemplate.png" Pack="true" PackagePath="" />
```

Comment:
this file will need to be physically moved into the images folder of the project. I only moved the SVG.

Reply:
the SVG could be moved back into the assets folder if no longer needed in the package.

Reply:
the PNG has already been moved 😄 I think it's best to keep them together rather than "assets" in two locations?

Reply:
I'm still only seeing the SVG in the images folder?

Reply:
also yeah could keep them both in images/

Reply:
Sorry, you're right! I must be blind 😄